example/helloworld/client.py

```diff
@@ -10,7 +10,33 @@
     channel = Channel(loop=asyncio.get_event_loop())
     stub = GreeterStub(channel)
 
-    print(await stub.SayHello(HelloRequest(name='World')))
+    # UNARY_UNARY - simple RPC
+    print(await stub.SayHello(HelloRequest(name='you')))
+    async with stub.SayHello.open() as stream:
```

Comment:
It would be great to visually divide simple method calls and more advanced usage using `stream` directly.

Reply:
I have visually separated the different method examples and the simple and more advanced usage examples within.